README.md

```diff
@@ -2,4 +2,4 @@
 
 Collection of open source sample extensions built on the forma extensions api.
 
-Docs: https://app.autodeskforma.com/extensions/preview/sdk.html
\ No newline at end of file
+Docs: https://app.autodeskforma.com/extensions/preview/sdk.html
```

Comment:
Dead link from "way-back" when analyze extensions had our own POC/preview SDK.

Should the samples directory just be top level instead? And remove the generators, data-providers and masterclass directories?